cot/templates/admin/model.html

```diff
@@ -34,6 +34,40 @@ <h2>{{ model.name() }}</h2>
 
     <footer>
         Displaying {{ objects.len() }} {{ model.name() }}{{ objects.len()|pluralize }}.
```

Comment:
I think we should also change this text to say something like "Displaying 10 out of 521 users" to make it clear that there are more on next pages.